### PR TITLE
Release 1.2.0

### PR DESCRIPTION
**Pull request by niomaster**
I would like to do a release soon, as I need a public .deb build for the docker image I'm building.

### CODE REVIEW

**Comment by raulmonti** [comment]
@niomaster Can we use this opportunity to review the readme? specifically the installation instructions?

**Comment by niomaster** [comment]
The installation instructions are correct for building from the git repository, but you're probably referring to #322 or #320?

**Comment by raulmonti** [comment]
Oh, good, but we can fix those two next time. I will review this now.

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+lazy val viper_api = (project in file("viper"))
+lazy val parsers = (project in file("parsers"))
+
+lazy val vercors = (project in file("."))
+    .dependsOn(viper_api)
+    .dependsOn(parsers)
+    .settings(
+        name := "Vercors",
+        organization := "University of Twente",
+        version := "1.1.0",
```
Comment (raulmonti) [inline build.sbt]: Do we need to change this to 1.2.0?

Comment (niomaster) [reply build.sbt]: Good catch, yes.

Comment (raulmonti) [reply build.sbt]: I guess this small change does not deserve a full travis test. So I will merge now.

**Pull request merged**